### PR TITLE
Adds v1.13.0 release notes

### PR DESCRIPTION
**Pull request by JoshVanL**
Adds `docs/release_notes/v1.13.0.md`.

### CODE REVIEW

**Review by yaron2 — DISMISSED** [review]

**Review by ItalyPaleAle — APPROVED** [review]

File: docs/release_notes/v1.13.0.md [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+
+> **Note: This release contains a few [breaking changes](#breaking-changes).**
+
+See [this](#upgrading-to-dapr-1.13) section on upgrading Dapr to version 1.13.
+
+## Highlights
+
+These are the v1.13 release highlights:
+
+### Component hot reloading
```
Comment (paulyuk) [inline docs/release_notes/v1.13.0.md:20]:
```suggestion
### Component hot reloading (preview)
```
Consider saying preview

File: docs/release_notes/v1.13.0.md [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+
+### Standardized error codes
+
+The Dapr PubSub and State APIs now have the appropriate and standardized error codes returned to applications including enriched error details based on gRPC's richer error model.
+The remaining APIs are a work in progress, and community contributions towards this effort are greatly appreciated and very impactful.
+See additional info on the error codes [here](https://docs.dapr.io/reference/api/error_codes/).
+Check the docs for each SDK for per-SDK error code parsing and handling.
+See an [example of the Go SDK error parsing and handling here](https://v1-13.docs.dapr.io/developing-applications/sdks/go/go-client/#error-handling).
+
+### Actor reminders performance improvement
```
Comment (paulyuk) [inline docs/release_notes/v1.13.0.md:54]: consider raising up the list.  perhaps after workflow?

File: docs/release_notes/v1.13.0.md [hunk 3]
```diff
@@ -0,0 +1,519 @@
...
+
+PostgreSQL has a new v2 implementation which contains improvements to performance and reliability.
+New applications are encouraged to use v2.
+The v1 implementation remains supported and is not deprecated.
+There is no migration path from v1 to v2.
+See additional info [here](https://v1-13.docs.dapr.io/reference/components-reference/supported-state-stores/setup-postgresql-v2/).
+
+#### Azure Blob Storage state store v2
+
+Azure Blob Storage has a new v2 implementation, which is recommended for all new projects as it fixes a backwards-incompatible bug with key prefixes.
```
Comment (paulyuk) [inline docs/release_notes/v1.13.0.md:88]: We should file a separate issue to upgrade docs and samples to lead with using these by default.

**Review by paulyuk — APPROVED** [review]
Minor comments

File: docs/release_notes/v1.13.0.md (hunk 3 above)
Comment (ItalyPaleAle) [reply docs/release_notes/v1.13.0.md:88]: Docs have been updated, do we use ABS somewhere in samples that you're aware of?

Comment (paulyuk) [reply docs/release_notes/v1.13.0.md:88]: Now that I look at it I see we don't have a doc on state management in Azure (which would use blobs etc), so I think we're good based on docs.  thank you!

File: docs/release_notes/v1.13.0.md (hunk 2 above)
Comment (JoshVanL) [reply docs/release_notes/v1.13.0.md:54]: :+1:

**Review by ItalyPaleAle — DISMISSED** [review]

**Review by mikeee — APPROVED** [review]

File: docs/release_notes/v1.13.0.md [hunk 4]
```diff
@@ -0,0 +1,519 @@
...
+> Note that while .NET 7 introduced polymorphic (de)serialization using `JsonDerivedType` attribute or `JsonPolymorphismOptions` on a custom `IJsonTypeInfoResolver` implementation, Dapr does not ship a .NET 7 target of the SDK.
+> Hence, a .NET 7 consumer application uses the .NET 6 target of the Dapr SDK.
+> Therefore, this breaking change only applies when upgrading a consumer application to .NET 8.
+
+> Note that the new serialization behavior in the .NET 8 version of the SDK allows for polymorphic deserialization of the response by a weakly-typed actor client.
+> This requires that the programmer specifies a `typeDiscriminator` on the `JsonDerivedType` attribute.
+
+### Java SDK
+
+- **RESOLVED** Update project-reactor dependency & fix deprecated calls [816](https://github.com/dapr/java-sdk/pull/816)
```
Comment (mikeee) [inline docs/release_notes/v1.13.0.md:519]: Are we using resolved for this tag?

Comment (yaron2) [reply docs/release_notes/v1.13.0.md:519]: Should be Updated

**Review by artursouza — APPROVED** [review]

**Pull request merged**